File: gameplay/js/unit_tests/messageLine.js
var catan = catan || {};
catan.models = catan.models || {};

/**
* messageLine Module
*
* @module catan.models.messageLine
*/

catan.models.messy = function messageLineNameSpace(){
	
	var messageLine = (function messageLineClass(){
		
		function messageLine()
		{
			this.message = "";
			this.source = "";
		}
		
		messageLine.prototype.setMessage = function(theMessage){
			this.message = theMessage;		
		}
	
		messageLine.prototype.getMessage = function getMessage(){
			return this.message;		
		}
			
		messageLine.prototype.setSource = function(theSource){
			this.source = theSource;		
		}
	
		messageLine.prototype.getSource = function(){
			return this.source;		
		}
		
		return messageLine;
	}());
	
	var myClass = (function myClassClass(){
		function myClass()
		{
			this.age = 8;		
		}
	}());
	return {messageLine:messageLine,
			myClass:myClass};
};

File: libs/DbPersist/src/server/api/utils/MessageLine.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package server.api.utils;

/**
 *
 * @author Wesley
 */
public class MessageLine {
	private String message;
	private String source;
	
	public MessageLine()
	{
		setMessage(null);
		setSource(null);
	}

	public MessageLine(String theMessage, String theSource){
		setMessage(theMessage);
		setSource(theSource);
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}

	public String getSource() {
		return source;
	}

	public void setSource(String source) {
		this.source = source;
	}
}
